tensorflow/tools/ci_build/builds/android.sh

```diff
@@ -21,4 +21,9 @@ source "${SCRIPT_DIR}/builds_common.sh"
 configure_android_workspace
 
 CPUS=armeabi-v7a,x86_64
-bazel build -c opt --fat_apk_cpu=${CPUS} //tensorflow/examples/android:tensorflow_demo
+
+# Enable sandboxing so that zip archives don't get incorrectly packaged
+# in assets/ dir (see https://github.com/bazelbuild/bazel/issues/2334)
+bazel --bazelrc=/dev/null build -c opt --fat_apk_cpu=${CPUS} \
```

Comment:
Could you add a TODO for me to revert these changes once sandboxing is enabled for all builds?